tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+import wazuh_testing.fim as fim
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+
+# Variables
+
+key = "HKEY_LOCAL_MACHINE"
+sub_key_1 = "SOFTWARE\\Classes\\testkey"
+sub_key_2 = "SOFTWARE\\Classes\\Testkey"
+
+registry_1, registry_2 = os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)
+wazuh_log_monitor = FileMonitor(fim.LOG_FILE_PATH)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+
+monitoring_modes = ['scheduled']
+
+# Configurations
+
+conf_params = {'WINDOWS_DUPLICATED_REGISTRY_1': registry_1,
+               'WINDOWS_DUPLICATED_REGISTRY_2': registry_2}
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_duplicated_registry.yaml')
+parameters, metadata = fim.generate_params(extra_params=conf_params, modes=monitoring_modes)
+configurations = load_wazuh_configurations(configurations_path, __name__,
+                                           params=parameters, metadata=metadata)
+
+registry_list = [(key, sub_key_1, fim.KEY_WOW64_32KEY), (key, sub_key_2,
+                 fim.KEY_WOW64_32KEY)]
```

Comment:
You have up to 120 characters. 
```suggestion
configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)

registry_list = [(key, sub_key_1, fim.KEY_WOW64_32KEY), (key, sub_key_2, fim.KEY_WOW64_32KEY)]
```

Reply:
Fixed in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/8d1fa4fe52a351600b58521da61f5af7e8b3400e)